projectDocs/community/expertsList.md

```diff
@@ -0,0 +1,23 @@
+# Community experts list
+
+This document outlines NV Access employees and community contributors.
+This can be used a reference to contact or tag developers in regards to relevant issues and pull requests.
+
+## NV Access
+
+For optimal responses, please avoid tagging NV Access employees on unrelated tickets.
+
+| GitHub | email | topics |
+|---|---|---|
+| @gerald-hartig | <gerald@nvaccess.org> | Controversial product decisions |
+| @seanbudd | <sean@nvaccess.org> | Issue triage, technical decisions |
+| @SaschaCowley | <sascha@nvaccess.org> | Issue triage, technical decisions |
+| @michaelDCurran | <mick@nvaccess.org> | Advanced technical consulting |
+| @Qchristensen | <quentin@nvaccess.org> | User documentation, user support |
+| @jcsteh | <jamie@nvaccess.org> | Firefox bugs |
```

Comment:
@jcsteh - confirming you're okay being added to this list, and can this be expanded to web browser bugs in general?

Reply:
Thanks for the ping.

1. While I'm gratified to be considered an honorary NV Access employee, this isn't really correct these days. :) I just don't want to send the wrong message here, so I should probably be moved to community now.
2. Even though I'm involved in web standards and I do have Chromium contacts, I don't have any direct influence over what other browsers do and I am not a Chromium contributor, so I think we should leave this as Firefox. If a bug ends up being in Firefox and other browsers, I will likely be involved to some extent in figuring out what all browsers should do anyway, but if it's purely a Chromium bug, I probably can't help much.